src/main/resources/db/migration/V1_18__alter_judicial_location_mapping_table.sql

```diff
@@ -0,0 +1,307 @@
+delete from judicial_location_mapping where judicial_base_location_id = '1030';
+
+INSERT INTO judicial_location_mapping (judicial_base_location_id,base_location_name,epimms_id,building_location_name) VALUES ('747','Royal Courts of Justice - High Court - Chancery','20262','ROYAL COURTS OF JUSTICE');
```

Comment:
you can have one single insert statement with multiple values
INSERT INTO <TableName> ( Column1, Column2 ) VALUES ( Value1, Value2 ), ( Value1, Value2 )

Reply:
thanks @SujeethKumarAri . These were generated through Excel using the formula given in the ticket and as there 305 records I didn't modify it, but if that's an issue I can change it.

Reply:
yes, I can understand @ishratt . The excel formula is just to prepare the statements. But we need to modify it for our performance
https://stackoverflow.com/questions/1793169/which-is-faster-multiple-single-inserts-or-one-multiple-row-insert#:~:text=If%20you%20are%20inserting%20many,separate%20single%2Drow%20INSERT%20statements.

Reply:
sure, in that case will definitely change it, thanks for the reference @SujeethKumarAri

Reply:
no problem @ishratt